### DISCUSSION TITLE
How can I run it in the background in Linux

### DISCUSSION THREAD

**Discussion by uguraltinsoy** [bluenviron/mediamtx#3138] [Questions]
### Question

How can I run it in the background in Linux?

I run it as "./mediamtx" but I want to run it in the background because I connect it to the raspberry pi with ssh and when I do "./mediamtx &" it even uses the terminal etc. I want it to run directly when the raspberry pi is turned on, so I added it to "rc.local" but I don't know if I am adding it correctly, how can I do it? 

```
GNU nano 5.4                                                /etc/rc.local                                                          
# This script is executed at the end of each multiuser runlevel.
# Make sure that the script will "exit 0" on success or any other
# value on error.
#
# In order to enable or disable this script just change the execution
# bits.
#
# By default this script does nothing.

# Print the IP address
_IP=$(hostname -I) || true
if [ "$_IP" ]; then
  printf "My IP address is %s\n" "$_IP"
fi

sudo /home/pi/mediamtx_v1/mediamtx &

exit 0
```

<img width="575" alt="Ekran Resmi 2024-03-12 21 12 23" src="https://github.com/bluenviron/mediamtx/assets/48494791/4eb31969-c118-4fca-a442-3edb43e0df72">

**Comment by aler9** [comment]
Instructions on how to run the server in the background are in the README:

https://github.com/bluenviron/mediamtx?tab=readme-ov-file#linux

[a comment by an automated account is not shown]
